### PR TITLE
PA Migration: Doc Updates

### PR DESCRIPTION
**Pull request by fpetrini15**
Point docs to new PA repo.

Server changes: https://github.com/triton-inference-server/server/pull/7488
Client changes: https://github.com/triton-inference-server/client/pull/771
PA changes: https://github.com/triton-inference-server/perf_analyzer/pull/21
DALI changes: https://github.com/triton-inference-server/dali_backend/pull/254
PyTriton changes: https://github.com/triton-inference-server/pytriton/pull/80
Tutorial changes: https://github.com/triton-inference-server/tutorials/pull/105

### CODE REVIEW

File: docs/config_search.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ It has two modes:
 
 The parameters that are automatically searched are
 [model maximum batch size](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#maximum-batch-size),
-[model instance groups](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups), and [request concurrencies](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/perf_analyzer.md#request-concurrency).
+[model instance groups](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups), and [request concurrencies](https://github.com/triton-inference-server/perf_analyzer/blob/main/docs/cli.md#measurement-options).
```
Comment (debermudez) [inline docs/config_search.md:101]: @nv-braf can you verify this line?

**Review by debermudez — COMMENTED** [review]
LGTM but i want @nv-braf to review that one link.

File: docs/config_search.md (hunk 1 above)
Comment (nv-braf) [reply docs/config_search.md:101]: Verified.

**Comment by nv-braf** [comment]
> LGTM but i want @nv-braf to review that one link.

Reviewed!

**Review by nv-braf — APPROVED** [review]

**Pull request merged**